website/root.css

```diff
@@ -0,0 +1,43 @@
+:root,
+body {
+  padding: 0;
+  margin: 0;
+  font-family: sans-serif;
+}
+
+:root {
+  color-scheme: light dark;
+
+  @media (prefers-color-scheme: light) {
+    background-color: #fff;
+    color: #111;
+  }
+
+  @media (prefers-color-scheme: dark) {
+    background-color: hsl(0deg 0% 10%);
+    color: #fff;
+  }
+}
+
+#root {
+  display: grid;
+  grid-template-columns: auto 1fr;
+}
+
+.rdg.fill-grid {
+  block-size: 100%;
+}
+
+.rdg.small-grid {
+  block-size: 300px;
+}
+
+.rdg.big-grid {
+  block-size: 600px;
+}
+
+.rdg-cell .Select {
```

Comment:
Is this used anywhere?

Reply:
I think it was used with `react-select` which we removed a long time ago?